integration_tests/__tests__/__snapshots__/failures-test.js.snap

```diff
@@ -62,7 +62,7 @@ Object {
     Expected value to be truthy, instead received
       false
       
-      at Object.throws (__tests__/assertion-count-test.js:14:17)
+      at throws (__tests__/assertion-count-test.js:14:17)
```

Comment:
Looking at the test, the inferred function name is `throws` and it's not attached to any `Object`, I believe this is actually more accurate that it previously was.